assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -240,9 +240,14 @@ che.infra.kubernetes.master_url=
 che.infra.kubernetes.trust_certs=
 
 # Defines the way how servers are exposed to the world in k8s infra.
-# List of  strategies implemented in Che: default-host, multi-host, single-host
+# List of strategies implemented in Che: default-host, multi-host, single-host
 che.infra.kubernetes.server_strategy=default-host
 
+# Defines the way in which the workspace plugins and editors are exposed in the single-host mode.
+# Supported exposures:
+# - 'native': Exposes servers using k8s Ingresses. Works only on Kubernetes.
```

Comment:
so nothing about gateway?

Reply:
it's not yet enabled, right ?

Reply:
You can set it, but it throws `UnsupportedOperationException` when exposing servers. So I'd rather keep it out of documentation for now. When `GatewayServerExposer` is implemented, I will of course write it here.